src/styles/stars.scss

```diff
@@ -0,0 +1,50 @@
+.stars {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 96px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: 50%;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &--1 {
+    .stars__star:nth-child(-n + 1) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--2 {
+    .stars__star:nth-child(-n + 2) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--3 {
+    .stars__star:nth-child(-n + 3) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--4 {
+    .stars__star:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--5 {
+    .stars__star:nth-child(-n + 5) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
Here you can use a loop to reduce the code. Please try to do it.

You can use `@for` or `@each` loop.